packages/styles/tooltip.css

```diff
@@ -156,7 +122,7 @@
 }
 
 .TooltipHead {
-  background: var(--tooltip-info-background-color);
+  background: var(--gray-20);
   font-size: 16px;
   color: var(--gray-90);
   padding: 16px 28px;
```

Comment:
Not a blocker since it's not part of your PR:

Only commenting because I noticed, but `font-size` and `padding` should really be using our cauldron vars.

Reply:
Created a ticket: https://github.com/dequelabs/cauldron/issues/707. We may want to just fix it when we update the big tooltip to support dark theme, since I'm guessing the designs will change a bit at that time anyway